src/main/resources/templates/list-GameList.html

```diff
@@ -23,7 +23,7 @@ <h4>All Console Games</h4>
 				<tr class="bg-danger">
 					<th>Name</th>
 					<th>Console</th>
-					<th>year</th>
+					<th>Year</th>
```

Comment:
Here I just capitalized the year table head of the game list table to make it consistent with the others / for aesthetics purposes.